core/src/main/resources/lib/hudson/newFromList/form.jelly

```diff
@@ -117,4 +117,4 @@ THE SOFTWARE.
     }
     updateOk(okButton.getForm());
   ]]></script>
-</j:jelly>
+</j:jelly>
```

Comment:
@kzantow Addressed here https://github.com/jenkinsci/jenkins/pull/2117/commits/5140598d316405c81087c2c86e3aa98b676e703e I don't know why/who changed this, maybe me ;(